### PR TITLE
Create blog_publishing_process.md

### LINKED ISSUE #364: Guide/process for publishing on Ghost
Guidance for publishing and using Ghost (metadata, ..)

### PR DESCRIPTION
**Pull request by CaroFG**
# Pull Request

## What does this PR do?
Fixes #364 

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: blog_publishing_process.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Blog publishing process
+
+## Context
+Once an article has been reviewed, authors need to follow the steps below to make sure their content is ready for publication.
+
+## Process
+1. Before publishing an article, you need access to [Ghost](https://meilisearch.ghost.io/ghost/)
+    - Meilisearch workers: ask someone from the DevRel team to create credentials for you
+    - External contributors: discuss this with your contact at Meilisearch
+2. Make sure you have properly filled your profile description. Check the [profile creation process](https://github.com/meilisearch/devrel/blob/main/process/blog_profile_creation.md) if needed.
```
Comment (ferdi05) [inline blog_publishing_process.md:10]: `filled in your`

File: blog_publishing_process.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        - Headings
+        - Font weight: bold, italics...
+        - Spaces between paragraphs
+        - Quotation marks
+        - Code snippets
+        - Lists
+        
+   <img src="https://user-images.githubusercontent.com/48251481/172438035-e9a83413-b1c2-4539-8b8c-f36341344011.png" width="25%"> 👉 <img src="https://user-images.githubusercontent.com/48251481/172438051-21a6acc8-9a44-4214-b4d0-ff6b012f7a83.png" width="25%">
+
+7. Insert the desired illustrations into the body (screenshots, etc.), please make sure they comply to the size and quality requirements.
```
Comment (ferdi05) [inline blog_publishing_process.md:26]: `they comply with the`

**Review by ferdi05 — CHANGES_REQUESTED** [review]
Superb work @CaroFG , congrats!
It's super clear, loving it!

A few changes on the grammar, mostly suggested by Grammarly, and then we're good to go!

**Comment by CaroFG** [comment]
I wanted to add the tags' step, but realized "regular" accounts don't have the possibility to add them @ferdi05

**Comment by ferdi05** [comment]
> I wanted to add the tags' step, but realized "regular" accounts don't have the possibility to add them @ferdi05

Do you mean that they can't select a tag with this role? or they can select an existing tag, but not create a new one?

**Pull request merged**